.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+jobs:
+  test:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 12
+          - 14
+          - 16
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Clone nodejs/node repository
+        uses: actions/checkout@v2
+        with:
+          repository: nodejs/node
+          path: tmp
```

Comment:
I'm not sure why we're cloning nodejs/node?

Reply:
The approach was similar to what we setup in remark-node-preset. The CI tests that new/modified rules still pass on the upstream code